conf/messages

```diff
@@ -25,6 +25,8 @@ caption = Transport details
 
 checkYourAnswers.heading = Check your answers
 checkYourAnswers.title = Check your answers - Transport details
+miniCheckYourAnswers.heading = Check your answers
+miniCheckYourAnswers.title = Check your answers - Transport details
```

Comment:
Content doc suggests this should be `Check your answers - Departure means of transport`

Reply:
And for the sake of consistency can these keys be something like `transportMeans.departure.checkYourAnswers` ? `miniCheckYourAnswers` is a bit vague